ara/cli/host.py

```diff
@@ -62,19 +78,17 @@ def get_parser(self, prog_name):
         parser.add_argument(
             'host',
             metavar='<host>',
-            help='Host name or id to show',
+            help='Host id to show',
         )
         return parser
 
     def take_action(self, args):
-        try:
-            host = (models.Host.query
-                    .filter((models.Host.id == args.host) |
-                            (models.Host.name == args.host)).one())
-        except models.NoResultFound:
+        host = models.Host.query.get(args.host)
```

Comment:
Oh, I see. That's different.  I was thinking that we could also support `host show <playbook> <hostname>`.  I think that something like `host show <hostname>` would always fail (because if you run a playbook more than once, you will have multiple matches for `<hostname>`).